src/frontend/calendar/index.js

```diff
@@ -29,6 +30,9 @@ function Calendar() {
 		setCalendarView,
 		setListView,
 	} = useViews();
+	if ( shouldForceListView && ! isListView() ) {
```

Comment:
we'll always have the flash, the HOC returns false initially… I tried swapping the logic, but that just did the reverse, and shows the list on large screens.

Reply:
Right, I looked into the underlying hook and it's the same problem. In the hook I cobbled together in https://github.com/Automattic/meeting-calendar/pull/44,  I was able to get it to not flash.

Reply:
We can go with your hook from #44 then, but I think it should default to the list view on small screens, rather than flipping around the html/css of the calendar table.